File: LearningProject/Models/Answer.cs
using System.ComponentModel.DataAnnotations;

namespace LearningProject.Models
{
public class Answer
    {
        public int Id { get; set; }
        public Question Question { get; set; }

        [Required(ErrorMessage = "Question ID is required")]
        public int QuestionId { get; set; }
        public string Name { get; set; }
        public bool IsCorrect { get; set; }
    }
}

File: client/src/app/core/models/Answer.ts
import { AnswerResult } from "./AnswerResult";
import { Question } from "./Question";

export interface Answer {
    id: number;
    question: Question | null;
    questionId: number;
    name?: string | null;
    isCorrect: boolean;
    answerResults?: AnswerResult[] | null;
  }
  